polar-core/src/vm.rs

```diff
@@ -741,6 +794,7 @@ impl PolarVirtualMachine {
         if self.log {
             self.print("⇒ backtrack");
```

Comment:
Should this be checking `self.polar_log`?

Reply:
This is the old log, which I haven't removed yet. I'm not logging backtracks for this new one because I didn't find them useful, rather I have the log indent based on subqueries which is a lot easier to follow.